src/main/java/frc/robot/Robot.java

```diff
@@ -219,7 +219,6 @@ public void robotPeriodic() {
                 "*** Auto cancelled in %.2f secs ***", Timer.getFPGATimestamp() - autoStart));
       }
       autoMessagePrinted = true;
```

Comment:
Was there a reason to remove the 'LEDs.getInstance().setAutoFinished(true)'?

Reply:
There were two different approaches for auto and the auto fade pattern. One relied on notification from the Robot class and the other was done internal to the LEDs class with the lastEnabledAuto and lastEnabledTime variables. I kept the latter and removed the former.